### ISSUE TITLE
Documentation for WebConfig example not working

### ISSUE THREAD

**Issue by bmcdavid** [codelibs/fess#906]
Following the example on http://fess.codelibs.org/11.0/admin/webconfig-guide.html for crawling FESS site, but using https://blog.wsol.com instead of example URL only indexes the first page.

If I remove the value in "Included URLs For Crawling", crawling occurs, but it goes beyond that domain and indexes www.wsol.com and many other sites, which i don't want.

Any suggestions?

**Comment by marevol** [comment]
Replace http://fess.codelibs.org/.* with https://blog.wsol.com/.*
http://fess.codelibs.org/11.0/admin/webconfig-guide.html#included-urls-for-crawling

**Comment by bmcdavid** [comment]
Sorry for being unclear earlier. But I did try that as show below. Also attaching the crawl log.
![crawl settings](https://cloud.githubusercontent.com/assets/2423177/23336544/608320b2-fba1-11e6-83ed-711b43033ab8.png)

[fess-crawler.zip](https://github.com/codelibs/fess/files/801691/fess-crawler.zip)

If I set the included urls for crawling to be blank, I do get results, the crawler just goes way beyond indexing the blog domain.

Thanks,
Brad
